butterfly-facade/src/main/java/com/paypal/butterfly/facade/Configuration.java

```diff
@@ -36,6 +37,20 @@ public Configuration(File outputFolder, boolean zipOutput) {
         setZipOutput(zipOutput);
     }
 
+    /**
+     * Butterfly default configuration
+     *
+     * @param outputFolder the output folder where the transformed application is
+     *                     supposed to be placed
+     * @param zipOutput if true, the transformed application folder will be compressed into a zip file
+     * @param modifyInPlace if true, the transformation will occur in application folder
+     */
+    public Configuration(File outputFolder, boolean zipOutput, boolean modifyInPlace) {
```

Comment:
No need to add a new constructor. If `-f` is present, then the default public constructor can be used instead.